lib/platform.py

```diff
@@ -152,6 +157,7 @@ def info():
     kernel = str(platform.release()),
     dist = str(linux_dist()),
     cpu = cpu,
+    mem = memory,
```

Comment:
This breaks the xml results that are generated in https://github.com/intel/vaapi-fits/blob/82765ec84ddc53bf32ec5ffb4662255177f0cf28/.slashrc#L301-L306

This `info` is passed as xml attributes to the xml output results.  And xml attribute values can only be string values.  The `mem` value here is an `int` and thus breaks the xml attribute rules.  I think the best fix would be to explicitly cast all attribute values in `info` to a `str` before passing it to the `et.Element`.

Reply:
Change `**info()` to `**{k: str(v) for k,v in info().items()}` in `et.Element` call should work.